### ISSUE TITLE
Failure to find all EFI binaries at Lenovo FL1 and/or FL2 update images

### ISSUE THREAD

**Issue by platomav** [chipsec/chipsec#1573]
Chipsec is not able to find all EFI images within some Lenovo "FL1" or "FL2" images when scanning for them via:

```bash
chipsec_main.py --skip_config --no_driver -n -m tools.uefi.scan_image -a generate $0AHJ000_FL2.json '/path/to/$0AHJ000.FL2'
```

The attached sample ($0AHJ000.FL2) can be used to reproduce the issue (after unzipping it):

[$0AHJ000.zip](https://github.com/chipsec/chipsec/files/9460795/0AHJ000.zip)

Opening the image in UEFITool, it consists of GUIDs “6C60EE00-C316-4C95-A684-CDC7E7033311” and “C8AB0F4E-26FE-40F1-9579-EA8D30D503A4” but Chipsec only detects “C8AB0F4E-26FE-40F1-9579-EA8D30D503A4”, which contains a Lenovo “flasher” image only, thus 1 result. However, “6C60EE00-C316-4C95-A684-CDC7E7033311” includes another 336 images which are not detected.

![UEFITool](https://user-images.githubusercontent.com/11527726/187662239-424fac2c-506a-4109-81f2-db2b9f815d6a.PNG)

The expected result is for Chipsec to detect 337 images from the given sample.

**Comment by npmitche** [comment]
Sorry for the long delay, but can you re-test using this PR: https://github.com/chipsec/chipsec/pull/1607

**Comment by platomav** [comment]
Thank you for the fix! I can verify that the attached sample now produces the expected amount of binaries, meaning 337 instead of 1. Feel free to close this when the PR is ready.

**Comment by npmitche** [comment]
Merged in #1607
